packages/template-retail-react-app/app/pages/checkout/partials/shipping-multi-address.test.js

```diff
@@ -71,13 +82,15 @@ jest.mock('@salesforce/retail-react-app/app/components/display-price', () => {
 const mockBasket = {
```

Comment:
@gurpreetsainisalesforce DisplayPrice Component is part of the rendering component. Why do we mock it? Can we not doing it? If you mock the data correctly, there is not reason to mock this component